src/templates/index.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+    <title>Auto-Reload on Content Change</title>
+</head>
+
+<body>
+    <h1>Waiting for voila.ipynb in input 1...</h1>
+    <p>If voilà doesn't load after connecting input 1, reload the Service UI</p>
+    <script>
+        function getContent() {
+            fetch('/') // Replace with your actual GET request URL
+                .then(response => response.text())
+                .then(data => {
+                    const hidden_element = document.getElementById('content');
+                    if (hidden_element) {
+                        const hidden_content = hidden_element.textContent;
+
+                        if (!data.includes(hidden_content) && data.includes(atob("dm9pbGE="))) {
```

Comment:
We might want to move this one to a constant
We also might want to use a better value to search for in here to make it more solid

Reply:
Done together.